editoast/src/infra_cache/mod.rs

```diff
@@ -556,6 +556,17 @@ impl InfraCache {
         }
         Ok(())
     }
+
+    pub fn get_route(&self, id: &str) -> Result<&Route> {
+        Ok(self
+            .routes()
+            .get(id)
+            .ok_or(InfraCacheEditoastError::ObjectNotFound {
+                obj_type: ObjectType::Route.to_string(),
+                obj_id: id.to_string(),
+            })?
```

Comment:
`.to_string()` causes allocations, therefore, lazyness on that kind of combinator is preferred.

```suggestion
            .ok_or_else(|| InfraCacheEditoastError::ObjectNotFound {
                obj_type: ObjectType::Route.to_string(),
                obj_id: id.to_string(),
            })?
```

Reply:
Done https://github.com/osrd-project/osrd/pull/5990/files#diff-fb61e16eab3011697c20a8dd2207aa73d56c484f64a982863c08577181ffc5a7R564
(github's "compare" doesn't help after rebase on dev and fixup :pray: )

Reply:
Yep, I noticed, 2 solutions:
- either push the _rebase_ independently
- push the fixup commits independently (then squash only after and push)